setup.py

```diff
@@ -36,9 +36,9 @@
                       'torch', 'einops', 'joblib', 'docstring_inheritance', 'torchinfo'],
     extras_require={
         'moabb': ['moabb'],
-        'tests': ['pytest', 'pytest-cov', 'codecov'],
+        'tests': ['pytest', 'pytest-cov', 'codecov', 'pytest_cases'],
```

Comment:
Could you also add pytest_cases in `requirement.txt` and docstring_inheritance in `docs/requirements.txt`?